lib/parse-message.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+//socket = one user, s (when in a loop) = all users
+
+module.exports = exports = {};
+
+exports.renameUserCommand = function (message, socket) {
+  socket.username = message.split(' ').slice(1).join(' ').trim();
+  socket.write(`You are now ${socket.username}`);
+}; //end of renameUserCommand function
+
+exports.directMessage = function (message, theOnlineUsersArray) {
```

Comment:
A ``/dm`` should also include the user's username prepended in the form of ``username: <message>``.